src/cascadia/TerminalControl/pch.h

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation.
+// Copyright (c) Microsoft Corporation.

```

Comment:
This should be a conflict, right? Adding a BOM is bad, removing one is a conflict?

Reply:
This file currently has a BOM. So this should be a removal. Maybe git ignores it during merge?
I'll merge main into this PR tomorrow just to be sure.